Pinta.Core/Actions/HelpActions.cs

```diff
@@ -47,10 +47,11 @@ public HelpActions ()
 			fact.Add ("Menu.Help.Translate.png", new Gtk.IconSet (PintaCore.Resources.GetIcon ("Menu.Help.Translate.png")));
 			fact.AddDefault ();
 
-			Website = new Gtk.Action ("Website", Catalog.GetString ("Pinta Website"), null, "Menu.Help.Website.png");
-			Bugs = new Gtk.Action ("Bugs", Catalog.GetString ("File a Bug"), null, "Menu.Help.Bug.png");
-			Translate = new Gtk.Action ("Translate", Catalog.GetString ("Translate This Application"), null, "Menu.Help.Translate.png");
-			About = new Gtk.Action ("About", Catalog.GetString ("About"), null, Stock.About);
+			// Translators: "Application name" (Pinta) + "Website"
```

Comment:
I've misunderstood what you were asking for here. 

Tried again but and thought I almost had it (something to do with Catalog.GetString marking it as a translatable string but I'm not getting it quite right) but will need to leave it until I've more time.